### PR TITLE
test-configs: d2500cc: block tinyconfig/allnoconfig

### PR DESCRIPTION
**Pull request by montjoie**
tinyconfig/allnoconfig does not work at all on d2500cc, block them.

### CODE REVIEW

**Comment by montjoie** [comment]
I see no global filter for allnoconfig/tinyconfig

**Comment by montjoie** [comment]
My d2500cc still receive lot of alnoconfig or tinyconfig.
Example: https://storage.kernelci.org/stable-rc/linux-4.9.y/v4.9.238-55-g132affe7fbd6/x86_64/allnoconfig/gcc-8/lab-clabbe/baseline-d2500cc.html

**Comment by gctucker** [comment]
> My d2500cc still receive lot of allnoconfig or tinyconfig.
> Example: https://storage.kernelci.org/stable-rc/linux-4.9.y/v4.9.238-55-g132affe7fbd6/x86_64/allnoconfig/gcc-8/lab-clabbe/baseline-d2500cc.html

Well yes, this PR hasn't been merged yet so it's not enabled in production.  But I see why now, the baseline test plan allows any config to be run except kselftest.

There is a default filter in `test-configs.yaml` for all the test plans:
```yaml
test_plan_default_filters:

  - combination: &arch_defconfig_filter
      keys: ['arch', 'defconfig']
      values:
        - ['arm', 'multi_v7_defconfig']
        - ['arm64', 'defconfig']
        - ['i386', 'i386_defconfig']
        - ['x86_64', 'x86_64_defconfig']

  - blocklist: &kselftest_defconfig_filter
      defconfig: ['kselftest']
```

so by default, only those defconfigs will get tested.  Then the baseline test plan overrides the filters to allow everything except kselftest:
```yaml
test_plans:

  baseline:
    rootfs: buildroot_baseline_ramdisk
    filters:
      - blocklist: *kselftest_defconfig_filter
```
and `tinyconfig` or `allnoconfig` get tested too.  However, I'm not seeing any jobs with these configs in lab-collabora.  I think this is because all the x86_64 devices in lab-collabora are blocking these configs, and no dtbs are built with them so arm/arm64 devices don't get tested either.

Basically, this PR does the same as every other x86_64 device listed in `test-configs.yaml` so it seems fine.  However, it would seem better to just add `tinyconfig` and `allnoconfig` to the block list filter of the baseline test plan and simplify things a bit that way.  Devices that _can_ run with such configs (not sure there are any) can override the filters and enable them if they want to.

**Comment by gctucker** [comment]
FYI I've built a `tinyconfig` x86_64 kernel on staging and no test jobs were generated for it, so the filter seems to be working:
```
+ cat /data/workspace/bot.staging.kernelci.org/test-runner/workspace/test-runner/artifacts/bmeta.json
{
    "arch": "x86_64",
    "build_environment": "gcc-8",
    "build_log": "build.log",
    "build_platform": [
        "Linux",
        "build-j3472-x86-64-gcc-8-tinyconfig-8bjrx",
        "4.15.0-1077-azure",
        "#82-Ubuntu SMP Thu Mar 26 03:25:09 UTC 2020",
        "x86_64",
        ""
    ],
    "build_threads": 10,
    "build_time": 29.61,
    "compiler": "gcc",
    "compiler_version": "8",
    "compiler_version_full": "gcc (Debian 8.3.0-6) 8.3.0",
    "cross_compile": "",
    "defconfig": "tinyconfig",
    "defconfig_full": "tinyconfig",
    "dtb_dir": null,
    "file_server_resource": "kernelci/staging-mainline/staging-mainline-20201020.0/x86_64/tinyconfig/gcc-8",
    "git_branch": "staging-mainline",
    "git_commit": "3177fa8a2caa54c77bf39f7b6fbf12ed1fe1000e",
    "git_describe": "staging-mainline-20201020.0",
    "git_describe_v": "v5.9-12064-g3177fa8a2caa",
    "git_url": "https://github.com/kernelci/linux.git",
    "job": "kernelci",
    "kconfig_fragments": "",
    "kernel_config": "kernel.config",
    "kernel_image": "bzImage",
    "kselftests": null,
    "modules": null,
    "status": "PASS",
    "system_map": "System.map",
    "text_offset": "0x01000000",
    "vmlinux_bss_size": 1417216,
    "vmlinux_data_size": 248192,
    "vmlinux_file_size": 11623880,
    "vmlinux_text_size": 4195641
}
```
https://staging.kernelci.org/build/id/5f8f610ac024b82bb03157b6/

**Review by gctucker — APPROVED** [review]
So basically, this is all fine :)  Merging now so this should get deployed in production tomorrow, then we'll be able to confirm whether the filters are actually working as intended in all cases.
